scripts/earthengine/earthengine_image.py

```diff
@@ -634,11 +612,18 @@ def ee_process(config) -> list:
         For supported params, refer to _DEFAULT_CONFIG.
         if ee_image_count > 1, then multiple images are exported with
         the start_time, end_time advanced by time_period.
+    Returns:
+      List of competed task status with gfs_file set to the image generated
+      if config['ee_wait_task'] is True, else a list of tasks launched.
     '''
     ee_tasks = []
     ee.Initialize()
     config['ee_image_count'] = config.get('ee_image_count', 1)
-    while config['ee_image_count'] > 0:
+    cur_date = utils.date_today()
+    counters = Counters()
+    # Get images by count or until yesterday
+    while (config['ee_image_count'] > 0) and (config.get('start_date', '') <
```

Comment:
nit: instead of `config['ee_image_count'] > 0` I think you can simply do: `config['ee_image_count']`

Reply:
Done